theme/styles/_config.scss

```diff
@@ -23,4 +23,13 @@ $additional-class-definitions: (
   'debug': (
     border: 1px solid red,
   ),
+  'safe-visibility-hidden': (
+    position: absolute,
+    text-indent: -10000px,
+    white-space: nowrap,
+    top: auto,
+    width: 1px,
+    height: 1px,
+    overflow: hidden,
+  )
```

Comment:
This is something we used on Tablet to fulfill the ADA requirement for an `h1` on every page.